local-kubernetes/local-isapi-kubernetes-manifest.yml

```diff
@@ -0,0 +1,65 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: gpitfutures-bc-deployment-isapi
+  namespace: buyingcatalogue
+spec:
+  replicas: 1
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+  minReadySeconds: 5
+  selector:
+    matchLabels:
+      app: gpitfutures-bc-deployment-isapi 
+  template:
+    metadata:
+      labels:
+        app: gpitfutures-bc-deployment-isapi
+    spec:
+      imagePullSecrets:
+      - name: regcredlocal
+      containers:
+      - name: gpitfutures-bc-isapi
+        image: gpitfuturesdevacr.azurecr.io/nhsd/buying-catalogue/identity-isapi
+        ports:
+        - containerPort: 80
+        resources:
+          requests:
+            cpu: 250m
+            memory: 100Mi
+          limits:
+            cpu: 1000m
+            memory: 500Mi
+        env:                                
+        - name: ASPNETCORE_ENVIRONMENT
+          value: Development
+        - name: CLIENTS__0__REDIRECTURLS__0
+          value: http://host.docker.internal:3000/oauth/callback
+        - name: CLIENTS__0__REDIRECTURLS__1
+          value: http://localhost:3000/oauth/callback
+        - name: CONNECTIONSTRINGS__CATALOGUEUSERS
+          value: Data Source=isapi-mssql-deployment.buyingcatalogue,1451;Initial Catalog=CatalogueUsers;MultipleActiveResultSets=True;User Id=NHSD;Password=DisruptTheMarket1!
+        - name: ISSUERURL
+          value: http://host.docker.internal:5102
```

Comment:
http://docker.for.win.localhost//?

Reply:
No need to change this file as its taken out of launch-env.. scripts.